lib/cisco_node_utils/command_reference_common_bgp.yaml

```diff
@@ -368,21 +368,15 @@ bgp_neighbor_af:
     config_set_append: '<state> as-override'
     default_value: false
 
-  cap_add_paths_receive:
+  additional_paths_receive:
     config_get_token_append: '/^capability additional-paths receive(?: disable)?/'
     config_set_append: '<state> capability additional-paths receive <disable>'
-    default_value: false
-
-  cap_add_paths_receive_disable:
-    default_value: false
+    default_value: 'inherit'
```

Comment:
I considered that but ultimately used strings just like Jie's code. Symbols are a bit puppet-centric anyway.